core/trino-main/src/test/java/io/trino/sql/planner/planprinter/TestJsonRepresentation.java

```diff
@@ -55,7 +55,7 @@
 
 public class TestJsonRepresentation
 {
-    private static final JsonCodec<JsonRenderedNode> JSON_RENDERED_NODE_CODEC = jsonCodec(JsonRenderedNode.class);
+    protected static final JsonCodec<JsonRenderedNode> JSON_RENDERED_NODE_CODEC = jsonCodec(JsonRenderedNode.class);
```

Comment:
There's no benefit in reusing this field across both tests. It just introduces unnecessary coupling. Keep it private and copy the line to the other test.